### DISCUSSION TITLE
How to disable extension on subdomains too? (exclude URLs)

### DISCUSSION THREAD

**Discussion by whoizit** [gdh1995/vimium-c#950] [Q&A] [answered]
There is a site that fights blockages by creating a new subdomain every day and when I open example.com from google search page, I get to the new subdomain. I need to disable this extension on whole site include subdomains, how can I do this?

example:
:https://26may.example.com/
:https://27may.example.com/
:https://28may.example.com/

this rule not works for me:
:https://([^\.]+\.)example.com/

**Comment by gdh1995** [comment, marked as answer]
A "Pattern" is a JS regular expression if it starts with "^", so you may try:
```
^https://[^.]+\.example\.com/
```
